### DISCUSSION TITLE
Cell voltage

### DISCUSSION THREAD

**Discussion by Salada71** [wills106/homeassistant-solax-modbus#776] [Ideas]
Is it planned to add support of cell temperature and voltage?
Registers 0x00BA...0x00BD
![obrazek](https://github.com/wills106/homeassistant-solax-modbus/assets/52708998/844abd01-1067-4c77-82ff-99c7a9c5fa5f)

Or is there possibility to add new registers by user? I tried it but without success. I am not programmer...

**Comment by wills106** [comment]
They are already exposed, but disabled by default.

**Reply by Salada71** [reply]
Oh, I see. Thanks.
